app/assets/stylesheets/application.scss

```diff
@@ -16,3 +16,78 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
The whole file is Spaces: 4. Please Change everything to Spaces: 2 like this picture.
![image](https://user-images.githubusercontent.com/35232348/95961850-37ad6500-0e38-11eb-9721-43eaaf9b5c6d.png)